lib/Backends/CPU/Transforms.cpp

```diff
@@ -79,6 +81,68 @@ static Node *optimizeCPUConv(ConvolutionNode *CN, Function *F) {
       CN->getBias(), CN->getKernels(), CN->getStrides(), CN->getPads(), group));
 }
 
+/// Convert a RWQFC node to its CPU version.
```

Comment:
Could you add a bit more explanation to this doxygen comment?  It's not immediately clear why we need a CPU-specific RowwiseQuantizedFC.  (I think I get it, but a bit of explanation would help :) ).